static/css/main.ie.css

```diff
@@ -1580,7 +1580,7 @@ textarea.success {
   display: inline;
   margin-right: 0;
   zoom: 1;
-  behavior: url('/static/vendor/box-sizing-polyfill/boxsizing.htc');
+  behavior: url('/src/vendor/box-sizing-polyfill/boxsizing.htc');
```

Comment:
Good catch, fixed in https://github.com/cfpb/cfgov-refresh/commit/4cb558a886af14613d0c1ef56af8515d0d50eae3 ... please scrutinize. Thx!!